storage/src/main/java/tech/pegasys/artemis/storage/MapDbDatabase.java

```diff
@@ -67,15 +73,58 @@
   private final ConcurrentNavigableMap<UnsignedLong, Set<Bytes32>> hotRootsBySlotCache =
       new ConcurrentSkipListMap<>();
 
+  private static void validateDatabaseVersion(File databaseVersionFile) {
+    try {
+      if (databaseVersionFile.exists()) {
+        Scanner contents = new Scanner(databaseVersionFile, UTF_8.name());
+        String ver = contents.nextLine();
+        if (StringUtils.isEmpty(ver) || !ver.equals(VERSION)) {
```

Comment:
Seems weird to explicitly check for empty.  If this is just avoiding nulls you could just use `!VERSION.equals(ver)`.